lms/djangoapps/verify_student/urls.py

```diff
@@ -139,4 +139,14 @@
         views.submit_photos_for_verification,
         name="verify_student_submit_photos"
     ),
+    # Endpoint for in-course reverification
+    # Users are sent to this end-point from within courseware
+    # to re-verify their identities by re-submitting face photos.
+    url(
+        r'^reverify/{course_id}/{check_point}/$'.format(
+            course_id=settings.COURSE_ID_PATTERN, check_point=settings.CHECK_POINT_PATTERN
```

Comment:
`check_point` should just be `checkpoint`. Similarly, `CHECK_POINT_PATTERN` can just be `CHECKPOINT_PATTERN`.